images/macos/templates/macOS-11-anka.pkr.hcl

```diff
@@ -0,0 +1,220 @@
+variable "source_vm_name" {
+    type = string
+    default = "clean_macos_11_300gb"
+  }
+  
+  variable "source_vm_tag" {
+    type = string
+    default = "bigsur_300g"
+  }
+  
+  variable "vm_name" {
+    type = string
+    default = "71"
+  }
+  
+  variable "vcpu_count" {
```

Comment:
Do we really want to define all these values as a variables?
May be define them directly to
```
  source "veertu-anka-vm-clone" "template" {
    vm_name = "${var.vm_name}"
    source_vm_name = "${var.source_vm_name}"
    source_vm_tag = "${var.source_vm_tag}"
    vcpu_count = "${var.vcpu_count}"
    ram_size = "${var.ram_size}"
    stop_vm =  "true"
  }
```
section?
Currently, It looks a bit massively.